packages/block-editor/src/components/tool-selector/index.js

```diff
@@ -67,19 +71,23 @@ function ToolSelector( props, ref ) {
 									value: 'edit',
 									label: (
 										<>
-											<Icon icon={ editIcon } />
-											{ __( 'Edit' ) }
+											{ selectIcon }
+											{ __( 'Design' ) }
 										</>
 									),
+									info: __(
+										'Full control over layout and styling.'
+									),
 								},
 								{
 									value: 'navigation',
 									label: (
 										<>
-											{ selectIcon }
-											{ __( 'Select' ) }
+											<Icon icon={ editIcon } />
+											{ __( 'Edit' ) }
```

Comment:
These are the largest changes in this PR.

@WordPress/gutenberg-design are you happy / confident with the naming conventions as these will be changed for _all_ users immediately when this PR is landed.

Reply:
The work around these modes has moved so quickly I'm not sure I'm fully up to speed on all the changes. 

With that disclaimer out of the way; one subjective observation is that I interpret some overlap between "Edit" and "Design" that may not be helpful... 'Design' feels to me like a _form_ of editing, so seeing the options presented at the same level feels a little off. 

I don't know if it would align with what the modes do, but I wonder if 'Edit' should be more aligned for the form of editing it is optimised for. E.g. there could be 'Design' and 'Write' modes.

Overall this feels like a change that needs more visibility. Making a decision in the depths of this PR might not be best?